generate_analysis_graphs.py

```diff
@@ -0,0 +1,106 @@
+import argparse
```

Comment:
What do you think the best way of integrating this into the flow will be? Adding it to the Docker run, or using a separate pipenv based script?

Reply:
Another docker-run script, to come in a future PR.